indigo/distribution.yaml

```diff
@@ -14645,6 +14645,17 @@ repositories:
       url: https://github.com/asmodehn/tornado-rosrelease.git
       version: 4.2.1-3
     status: maintained
+  tr_hub_parser:
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/Terabee/tr_hub_parser.git
```

Comment:
The release repository should be separate from the source repository. Please use [bloom](http://wiki.ros.org/bloom) to populate the release repo.